website/docs/docs/cloud/git/connect-gitlab.md

```diff
@@ -38,6 +40,13 @@ When you click **Link**, you will be redirected to GitLab and prompted to sign i
 
 Once you've accepted, you should be redirected back to <Constant name="cloud" />, and you'll see that your account has been linked to your profile.
 
+### Limitations
+
+Because <Constant name="cloud" /> Team plans use a single GitLab deploy token that's tied to the user who connected the repo, they are limited in the following ways when using GitLab:
```

Comment:
```suggestion
Team and Developer plans only allow single-user commit access. 

Because <Constant name="cloud" /> Team plans use a single GitLab deploy token that's tied to the user who connected the repo, they are limited in the following ways when using GitLab:
```